### PR TITLE
[ja] Translate tasks/administer-cluster/kubeadm/kubeadm-certs into Japanese

### PR DESCRIPTION
**Pull request by ptux**
Address #31173

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
/assign @atoato88

File: content/ja/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+CSRとは、クライアント用の署名付き証明書をCAに要求することを表します。
+kubeadmの用語では、通常ディスク上のCAによって署名される証明書をCSRとして生成することができます。しかし、CAはCSRとして生成することはできません。
+
+### 証明書署名要求の作成 (CSR)
+
+`kubeadm certs renew --csr-only`で証明書署名要求を作成することができます。
+
+CSRとそれに付随する秘密鍵の両方が出力されます。
+ディレクトリを`--csr-dir`で渡すと、指定した場所にCSRを出力することができます。
+csr-dir`を指定しない場合は、デフォルトの証明書ディレクトリ(`/etc/kubernetes/pki`)が使用されます。
```
Comment (atoato88) [inline content/ja/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md:188]: missing ` at the begining.

```suggestion
`csr-dir`を指定しない場合は、デフォルトの証明書ディレクトリ(`/etc/kubernetes/pki`)が使用されます。
```

File: content/ja/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+
+証明書は`kubeadm certs renew --csr-only`で更新することができます。
+`kubeadm init`と同様に、`--csr-dir`フラグで出力先ディレクトリを指定することができます。
+
+CSRには、証明書の名前、ドメイン、IPが含まれますが、用途は指定されません。
+証明書を発行する際に、[正しい証明書の使用法](/docs/setup/best-practices/certificates/#all-certificates)を指定するのはCAの責任です。
+
+* `opensl`では、[`openssl ca`コマンド](https://superuser.com/questions/738612/openssl-ca-keyusage-extension)を使って行います。
```
Comment (atoato88) [inline content/ja/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md:196]:
```suggestion
* `openssl`では、[`openssl ca`コマンド](https://superuser.com/questions/738612/openssl-ca-keyusage-extension)を使って行います。
```

File: content/ja/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md [hunk 3]
```diff
@@ -0,0 +1,268 @@
...
+```
+
+デフォルトでは、これらのサービング証明書は1年後に失効します。
+
+Kubeadmは`KubeletConfiguration`フィールド`rotateCertificates`を`true`に設定します。これは有効期限が切れる間際に、サービング証明書のための新しいCSRセットを作成し、ローテーションを完了するために承認する必要があることを意味します。
+
+詳しくは[Certificate Rotation] (/docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping/#certificate-rotation)をご覧ください。
+
+これらのCSRを自動的に承認するためのソリューションをお探しの場合は、以下をお勧めします。
+クラウドプロバイダーに連絡し、CSRの署名者がいるかどうか尋ねてください。
```
Comment (atoato88) [inline content/ja/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md:260]:
```suggestion
クラウドプロバイダーに連絡し、ノードの識別をアウトオブバンドのメカニズムで行うCSRの署名者がいるかどうか尋ねてください。
```

File: content/ja/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md [hunk 4]
```diff
@@ -0,0 +1,268 @@
...
+ノードの識別をアウトオブバンドのメカニズムで行います。
```
Comment (atoato88) [inline content/ja/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md:261]:
```suggestion
```

**Review by atoato88 — COMMENTED** [review]
Almost LGTM, I left some comments about document format and typos.
Please check those.

File: content/ja/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md [hunk 5]
```diff
@@ -0,0 +1,269 @@
...
+承認するためには、次のようにします:
+```shell
+kubectl certificate approve <CSR-name>
+```
+
+デフォルトでは、これらのサービング証明書は1年後に失効します。
+
+Kubeadmは`KubeletConfiguration`フィールド`rotateCertificates`を`true`に設定します。これは有効期限が切れる間際に、サービング証明書のための新しいCSRセットを作成し、ローテーションを完了するために承認する必要があることを意味します。
+
+詳しくは[Certificate Rotation] (/docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping/#certificate-rotation)をご覧ください。
```
Comment (atoato88) [inline content/ja/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md:259]:
```suggestion
詳しくは[Certificate Rotation](/docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping/#certificate-rotation)をご覧ください。
```

File: content/ja/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md [hunk 6]
```diff
@@ -0,0 +1,269 @@
...
+
+これらのCSRを自動的に承認するためのソリューションをお探しの場合は、以下をお勧めします。
+クラウドプロバイダーに連絡し、ノードの識別をアウトオブバンドのメカニズムで行うCSRの署名者がいるかどうか尋ねてください。
+
+{{% thirdparty-content %}}
```
Comment (atoato88) [inline content/ja/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md:264]: I checked [preview site for this PR](https://deploy-preview-31213--kubernetes-io-main-staging.netlify.app/ja/docs/tasks/administer-cluster/kubeadm/kubeadm-certs/).
The last section of [here](https://deploy-preview-31213--kubernetes-io-main-staging.netlify.app/ja/docs/tasks/administer-cluster/kubeadm/kubeadm-certs/#kubelet-serving-certs) has English text about "thirdparty-content" as is. (just before "Feedback" section.)
Could you localize those texts?
I think target file for those is [`data/i18n/ja/ja.toml`](https://github.com/kubernetes/website/blob/main/data/i18n/ja/ja.toml) and original English texts are on [`data/i18n/en/en.toml`](https://github.com/kubernetes/website/blob/main/data/i18n/en/en.toml#L234-L245).

File: data/i18n/ja/ja.toml [hunk 7]
```diff
@@ -184,6 +184,12 @@ other = "購読する"
 [ui_search_placeholder]
 other = "検索"
 
+[thirdparty_message]
+other = """このセクションでは、Kubernetesが必要とする機能を提供するサードパーティープロジェクトにリンクしています。これらのプロジェクトがアルファベット順に記載されていて、Kubernetesプロジェクトの作者はに責任を持ちません。このリストにプロジェクトを追加するには、変更を提出する前に<a href="/docs/contribute/style/content-guide/#third-party-content">content guide</a>をお読みください。<a href="#third-party-content-disclaimer">詳細はこちら。</a>"""
```
Comment (atoato88) [inline data/i18n/ja/ja.toml:188]:
```suggestion
other = """このセクションでは、Kubernetesが必要とする機能を提供するサードパーティープロジェクトにリンクしています。これらのプロジェクトがアルファベット順に記載されていて、Kubernetesプロジェクトの作者は責任を持ちません。このリストにプロジェクトを追加するには、変更を提出する前に<a href="/docs/contribute/style/content-guide/#third-party-content">content guide</a>をお読みください。<a href="#third-party-content-disclaimer">詳細はこちら。</a>"""
```

**Comment by atoato88** [comment]
Thank you for quick update!! :smile: 
/lgtm

**Pull request merged**